src/lib/Syntax.hs

```diff
@@ -186,6 +186,9 @@ instance Semigroup (LabeledItems a) where
   LabeledItems items <> LabeledItems items' =
     LabeledItems $ M.unionWith (<>) items items'
 
+instance Monoid (LabeledItems a) where
+  mempty = NoLabeledItems
```

Comment:
Missing `mappend`?

Reply:
`mappend` defaults to `(<>)` which is defined above.